crates/e2e/macro/Cargo.toml

```diff
@@ -21,10 +21,11 @@ proc-macro = true
 
 [dependencies]
 ink_ir = { version = "4.0.0-beta", path = "../../ink/ir" }
+contract-build = "2.0.0-beta"
```

Comment:
So we can either pull in a Git dep here so that we can use `+nightly` again, or wait until a new version of `cargo-contract` is released. I'm okay with waiting for a new version of `cargo-contract`

Reply:
Updated to the latest `2.0.0-beta.1` release which allows `+nightly`. Also `--features` can be passed through now anyway from the command line so you can also do `cargo contract test --features e2e-tests`